### PR TITLE
add msg to address

### PR DESCRIPTION
**Pull request by hchoura**

### CODE REVIEW

File: src/foam/nanos/auth/Address.js [hunk 1]
```diff
@@ -22,16 +22,24 @@ foam.CLASS({
   ],
 
   messages: [
-    { name: 'INVALID_POSTAL_CODE_ERR_MSG', message: 'Invalid postal code' }
+    { name: 'INVALID_POSTAL_CODE_ERR_MSG', message: 'Invalid postal code' },
+    { name: 'INVALID_ADDRESS_1', message: 'Invalid value for address 1.' },
+    { name: 'PROVINCE_REQUIRED', message: 'Province required.' },
```
Comment (kgrgreer) [inline src/foam/nanos/auth/Address.js:27]: Should this be REGION_REQUIRED?

File: src/foam/nanos/auth/Address.js [hunk 2]
```diff
@@ -128,11 +136,11 @@ foam.CLASS({
         if ( typeof regionId !== 'string' || regionId.length === 0 ) {
           switch ( countryId ) {
             case 'CA':
-              return 'Province required.';
+              return this.PROVINCE_REQUIRED;
```
Comment (kgrgreer) [inline src/foam/nanos/auth/Address.js:139]: This doesn't look correct. We should just be using this.REGION_REQUIRED and let the normal i18n system replace that message with the appropriate one.

Comment (jlhughes) [reply src/foam/nanos/auth/Address.js:139]: Yes, all this should be removed now.  English - we can use region or province/state (to handle CA/US)  and then i18n for the rest.

**Comment by jlhughes** [comment]
Addressed in another PR.

**Pull request closed without merging**